dev/cognitiveservices/data-plane/Language/analyzetext-authoring.json

```diff
@@ -3174,6 +4253,23 @@
           "format": "float",
           "description": "The threshold of the class with the highest confidence, at which the prediction will automatically be changed to \"None\". The value of the threshold should be between 0 and 1 inclusive.",
           "type": "number"
+        },
+        "amlProjectPath": {
+          "description": "The path to the AML connected project.",
```

Comment:
What format is this path? Is it a URI as used above? I'd be consistent.

Reply:
It is the relative path for the project inside the AML container.